core/src/main/java/org/elasticsearch/search/sort/GeoDistanceSortBuilder.java

```diff
@@ -60,6 +79,29 @@ public GeoDistanceSortBuilder(String fieldName) {
     }
 
     /**
+     * Copy constructor.
+     * */
+    public GeoDistanceSortBuilder(GeoDistanceSortBuilder original) {
```

Comment:
Maybe package private if only needed in the tests like at the moment? Or move there? No strong opinions though.